reporter-web-app/src/models/WebAppPackage.js

```diff
@@ -336,6 +342,10 @@ class WebAppPackage {
         return this.#curations;
     }
 
+    get authors() {
```

Comment:
Property functions are alphabetically sorted so authors() should be in front of binaryArtifact()